subsys/bluetooth/controller/hci_driver.c

```diff
@@ -644,6 +655,13 @@ static int configure_supported_features(void)
 		}
 	}
 
+	if (IS_ENABLED(CONFIG_BT_CTLR_SDC_PAWR_ADV)) {
```

Comment:
I'd prefer if we used #if defined(CONFIG_BT_CTLR_SDC_PAWR_ADV) here too

Reply:
`IS_ENABLED` is used for all the others. The call to `sdc_support_le_periodic_adv_with_rsp` is optimized away and the linker will not include anything PAwR-related unless enabled.